### PR TITLE
Add numpy as `np` to console predefined variables

### PR DESCRIPTION
**Pull request by Czaki**
# Description

Add `np` as `numpy` alias to console variables

### CODE REVIEW

**Review by brisvag — APPROVED** [review]
I like this. I'm sure this came uo before and some folks were against polluting the namespace with out own stuff, but I feel that `np` is pretty much always needed anyways.

**Review by psobolewskiPhD — APPROVED** [review]
This is very magic, but it's also super convenient...
🥶

**Comment by jni** [comment]
Could I get @Carreau to look at reviving napari/napari-console#18 again? 🙏 In the meantime, I'm ok with this as a stopgap.

**Comment by Czaki** [comment]
> Could I get @Carreau to look at reviving [napari/napari-console#18](https://github.com/napari/napari-console/pull/18) again? 🙏 In the meantime, I'm ok with this as a stopgap.

but this is independent to this PR. Putting np was requested during hackaton when we just launch viewer (`napari` command) and test multiple things.

**Comment by jni** [comment]
I don't think it's independent — I don't think we should have np *or* viewer if we have the enclosing environment. e.g. some users might use numpy without renaming it. We shouldn't pollute their namespace with random stuff. But again, happy for this to go in.

**Comment by Czaki** [comment]
Still, if someone just launches napari as an application, not from script/notebook, then we should control the namespace.

**Comment by Carreau** [comment]
> Could I get @Carreau to look at reviving [napari/napari-console#18](https://github.com/napari/napari-console/pull/18) again? 🙏 In the meantime, I'm ok with this as a stopgap.

Yes, we can resume it, I think the needed patches id napari have been in napari long enough.

**Comment by brisvag** [comment]
> Still, if someone just launches napari as an application, not from script/notebook, then we should control the namespace.

Yeah I agree, in that case there is nothing to pollute, and it's the only case where this matters since from script/notebook people will probably already have all they want/need preimported

File: napari/_qt/qt_viewer.py [hunk 1]
```diff
@@ -1136,3 +1140,19 @@ def _create_remote_manager(
...
 
     return manager
+
+
+def _in_napari(n: int, frame: FrameType):
+    """
+    Predicates that return Wether we are in napari by looking
+    at:
+        1) the frames modules names:
+        2) the min_depth
```
Comment (brisvag) [inline napari/_qt/qt_viewer.py]:
```suggestion
    Determines whether we are in napari by looking at:
        1) the frames modules names:
        2) the min_depth
```

**Review by brisvag — APPROVED** [review]

**Pull request merged**